chart/values.yaml

```diff
@@ -2494,14 +2494,7 @@ dags:
     #
     # If you are using an ssh clone url, you can load
     # the ssh private key to a k8s secret like the one below
-    #   ---
```

Comment:
Don't remove the example. Not everyone will want to include the key in their values file.

Reply:
reverted